cmd/stack_package_test.go

```diff
@@ -375,16 +365,11 @@ func TestPackageNoTemplates(t *testing.T) {
 	log.InitLogging(&outBuffer, &outBuffer)
 
 	stackDir := filepath.Join(cmdtest.TestDirPath, "starter")
-	var err error
-	if runtime.GOOS == "windows" {
-		err = cmd.CopyDir(log, stackDir, (filepath.Join(sandbox.ProjectDir, "starter")))
-	} else {
-		err = cmd.CopyDir(log, stackDir, sandbox.ProjectDir)
-	}
+	err := cmd.CopyDir(log, stackDir, (filepath.Join(sandbox.ProjectDir, "starter")))
 	if err != nil {
-		t.Errorf("Problem copying %s to %s: %v", stackDir, sandbox.ProjectDir, err)
+		t.Errorf("Problem copying %s to %s: %v", stackDir, (filepath.Join(sandbox.ProjectDir, "starter")), err)
```

Comment:
Just a suggestion: we could set a variable for the `filepath.Join(...)` and then use is where needed. This could be applied to all these similar cases.